spec/models/video_spec.rb

```diff
@@ -4,42 +4,37 @@
   it { should belong_to(:category)}
   it { should validate_presence_of(:title)}
   it { should validate_presence_of(:description)}
+  it { should have_many(:reviews).order("created_at DESC")}
 
   describe "search by title" do 
```

Comment:
When you describe a method - you should prefix it with either `.` if its class method or `#` if its instance method. Check this here - http://www.betterspecs.org/#describe